src/themes/material/templates/edit-cell-array.tpl.html

```diff
@@ -2,9 +2,7 @@
 	<div class="q-grid-editor q-grid-array">
 		<h2 class="md-title">Edit {{$cell.column.title}}</h2>
 		<div class="q-grid-array-view">
-			<md-chip-list>
-				<md-chip *ngFor="let item of $cell.value" selected="true">{{item}}</md-chip>
-			</md-chip-list>
+			<q-grid-chips [cell-value]="$view.edit.cell.value"></q-grid-chips>
```

Comment:
cell-value to ng-model or items? chips should be independed from cell, even for names